### DISCUSSION TITLE
How To Add Theme On SearXNG

### DISCUSSION THREAD

**Discussion by sagarbhusal0** [searxng/searxng#3707] [Q&A] [answered]
Hello, I want to add a theme to my SearxNG VPS. I need your help with this. I’m looking for a theme similar to or more beautiful than the one on [Nibblehole](https://nibblehole.com/).

**Comment by Pietro395** [comment]
You can apply themes client side with the Stylus extension

An example here:
https://github.com/catppuccin/userstyles/tree/main/styles/searxng

**Reply by ItzELECTR0** [reply]
> You can apply themes client side with the Stylus extension

What if I want server side? (I'm self hosting and I want to push a custom theme locally)

**Reply by Qbsoon** [reply]
> > You can apply themes client side with the Stylus extension
> 
> What if I want server side? (I'm self hosting and I want to push a custom theme locally)

Exactly. This is the real issue. There should be a way for server-side themes and styles extension

**Comment by Bishop-Fox** [comment]
Safari stylus app

**Comment by return42** [comment, marked as answer]
Ceate your fork and customize https://github.com/searxng/searxng/blob/master/client/simple/src/less/definitions.less
